app/src/main/java/org/oppia/android/app/customview/MarqueeView.kt

```diff
@@ -0,0 +1,236 @@
+package org.oppia.android.app.customview
+
+import android.content.Context
+import android.graphics.Paint
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.animation.Animation
+import android.view.animation.Interpolator
+import android.view.animation.LinearInterpolator
+import android.view.animation.TranslateAnimation
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import android.widget.ScrollView
+import android.widget.TextView
+import java.lang.RuntimeException
+import kotlin.math.abs
+
+/**
+ * Provides a simple marquee effect for a single [android.widget.TextView].
+ * Reference: https://github.com/ened/Android-MarqueeView/blob/master/library/src/main/java/asia/ivity/android/marqueeview/MarqueeView.java
```

Comment:
We can't just copy an existing view (or any code) from a third-party. If we want to use MarqueeView but need to change it, then we should fork the original repo, make the changes we need, and add it as a dependency (using jitpack for simplicity). We actually do this for other situations, such as KotliTeX and AndroidSVG.

Reply:
Makes sense i will need to fork this and update the library. Is it okay if i fork to my account or it will need to be under the Oppia Project.

Reply:
I have made a fork here which you can use: https://github.com/oppia/Android-MarqueeView